constellations/gemini-admin/src/home/home.js

```diff
@@ -1,108 +1,110 @@
+import { HomeTabSectionClasses } from "./homeClasses.js";
+import { ClassesNotif } from "./homeClasses.js";
+import { HomeTabSectionUsers } from "./homeUsers.js";
+import { HomeTabSectionRegistrations } from "./homeRegistrations.js";
+import { HomeTabSectionAccounts } from "./homeAccounts.js";
+
+const sectionDisplayNames = {
+    class: "Unpublished Classes",
+    user: "New Users",
+    registration: "New Registrations",
+    unpaid: "Unpaid Accounts",
+};
 
 export class HomePage extends React.Component {
     state = {
-        classes: [],
+        currentSection: "class",
     };
 
-    componentDidMount() {
-        this.fetchData();
-    }
-
-    fetchData = () => {
-        API.get("api/unpublished").then((res) => {
-            const unpublishedList = res.data;
-            this.setState({
-                classes: unpublishedList.classes,
-            });
+    changeSection = (sectionName) => {
+        this.setState({
+            currentSection: sectionName,
         });
     };
 
     render() {
-        let unpublishedClasses = this.state.classes.map((row, index) => {
-            return <li key={index}> {row.classId} </li>;
-        });
+        var sectionComponent = <div></div>;
+
+        if (this.state.currentSection == "class") {
+            sectionComponent = <HomeTabSectionClasses />;
+        } else if (this.state.currentSection == "user") {
+            sectionComponent = <HomeTabSectionUsers />;
+        } else if (this.state.currentSection == "registration") {
+            sectionComponent = <HomeTabSectionRegistrations />;
+        } else {
+            sectionComponent = <HomeTabSectionAccounts />;
+        } //unpaid account
 
         return (
             <div id="view-home">
                 <h1>Administrator Dashboard</h1>
 
-                <div className="section">
-                    <div className="container-class">
-                        <h3 className="section-header">Unpublished Classes</h3>{" "}
-                        <button id="publish">
-                            <Link to={"/classes"}>View Details to Publish</Link>
-                        </button>
-                    </div>
-
-                    <div className="class-section">
-                        <div className="list-header">Class ID</div>
-                        <ul>{unpublishedClasses}</ul>
-                    </div>
+                <div className="tabs">
+                    <TabButton
+                        onChangeTab={this.changeSection}
+                        highlight={this.state.currentSection == "class"}
+                        section={"class"}
+                        buttonNum={<ClassesNotif />}
+                    />
+                    <TabButton
+                        onChangeTab={this.changeSection}
+                        highlight={this.state.currentSection == "user"}
+                        section={"user"}
+                        //buttonNum = {<UsersNotif />}cnewUsers.length in homeUsers
```

Comment:
I think this is correct. The reason buttonNum is always 0 is because you need it to be a certain value.
`buttonNum` from what it looks like is supposed to be an integer.

So you can simply do something like this: `buttonNum = {10}` or `buttonNum={newUsers.length}`